pkg/apis/api.acorn.io/v1/event.go

```diff
@@ -0,0 +1,47 @@
+package v1
+
+import (
+	"time"
+
+	internalv1 "github.com/acorn-io/runtime/pkg/apis/internal.acorn.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type Event internalv1.EventInstance
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type EventList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []Event `json:"items"`
+}
+
+// Alias helper types so that clients don't need to import the internal package when creating events.
+
+// +k8s:deepcopy-gen=false
+
+type EventResource = internalv1.EventResource
+
+// +k8s:deepcopy-gen=false
+
+type MicroTime = internalv1.MicroTime
```

Comment:
Related: At this point, it's very unlikely that any cluster still has events stored with `metav1.Time` formatted `observed` timestamps -- it's probably safe to drop the `internalv1.MicroTime` named type across the board in favor of `metav1.MicroTime`.